hack/sandbox.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Ensure kind is installed
+if ! command -v kind >/dev/null; then
+  echo "kind is not installed 😥. Please install kind before running this script."
+  exit 1
+fi
+
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+EOF
+echo
+echo
+echo "Loading polytomic image 📦 ..."
+# Load polytomic docker image into kind
+kind load docker-image polytomic-jake
```

Comment:
This hack is because I had to rebuild an arm64 container to load (using M1 chip). Otherwise, this can be polytomic image